### PR TITLE
Update redgate/flyway Readme

### PR DESCRIPTION
**Pull request by alistair-RG**
- Change links to flywaydb.org
- Add info about Flyway service
- Improve volume mounting palaver with workingDirectory instead

### CODE REVIEW

File: enterprise/README.md [hunk 1]
```diff
@@ -39,20 +24,34 @@ This will give you Flyway Command-line's usage instructions.
 
 To do anything useful however, you must pass the arguments that you need to the image. For example:
 
-`docker run --rm redgate/flyway -licenseKey="FL01..." -url=jdbc:h2:mem:test -user=sa info`
+`docker run --rm redgate/flyway -licenseKey="FL01..." -url=sqlite:/flyway/project/dev.db info`
```
Comment (WilliamL-RG) [inline enterprise/README.md]: this url needs `jdbc:` at the start and should be in the same format as other url uses

File: enterprise/README.md [hunk 2]
```diff
@@ -1,33 +1,18 @@
-# Official Flyway Teams and Enterprise Docker images
+# Official Flyway Community, Teams and Enterprise Docker images
 
-[![Docker Auto Build](https://img.shields.io/docker/cloud/automated/redgate/flyway)][docker]
+This is the official repository for [Flyway Command-line](https://documentation.red-gate.com/fd/welcome-to-flyway-184127914.html) images.
 
-[docker]: https://hub.docker.com/r/redgate/flyway/
```
Comment (piers-williams) [inline enterprise/README.md:5]: This line was kept in the OSS page

File: enterprise/README.md [hunk 3]
```diff
@@ -1,33 +1,18 @@
...
-To make it easy to run Flyway the way you want to, the following volumes are supported:
-
-Volume            | Usage
+Folder            | Usage
 ------------------|------
-`/flyway/conf`    | Directory containing a `flyway.conf` [configuration file](https://flywaydb.org/documentation/usage/commandline/#configuration)
-`/flyway/drivers` | Directory containing the [JDBC driver for your database](https://flywaydb.org/documentation/usage/commandline/#jdbc-drivers)
-`/flyway/sql`     | The SQL files that you want Flyway to use (for [SQL-based migrations](https://flywaydb.org/documentation/concepts/migrations#sql-based-migrations))
-`/flyway/jars`    | The jars files that you want Flyway to use (for [Java-based migrations](https://flywaydb.org/documentation/concepts/migrations#java-based-migrations))
+`/conf`    | Directory containing a `flyway.conf/toml` [configuration file](https://documentation.red-gate.com/fd/configuration-files-224003079.html)
```
Comment (piers-williams) [inline enterprise/README.md]:
```suggestion
`/flyway/conf`    | Directory containing a `flyway.conf/toml` [configuration file](https://documentation.red-gate.com/fd/configuration-files-224003079.html)
```

Should these remain `/flyway/x` like the OSS image?

Comment (piers-williams) [inline enterprise/README.md]:
```suggestion
`/conf`    | Directory containing a `flyway.[conf|toml]` [configuration file](https://documentation.red-gate.com/fd/configuration-files-224003079.html)
```

it's common to express options in this format - slashes are potentially a bit confusing in a filepath

File: enterprise/README.md (hunk 2 above)
Comment (alistair-RG) [reply enterprise/README.md:5]: I know - but I have no idea what it does. I suppose the safe thing is to keep it for now

File: enterprise/README.md (hunk 3 above)
Comment (alistair-RG) [reply enterprise/README.md]: It was an oversight on the flyway/flyway page - originally these were individual volume mounts but it makes it ungainly to use docker & flyway that way. The recent workingdirectory param updates should mean you just mount the project folder and go from there which is more in line with Desktop. I'm wondering if this ought to be pulled out of here and put into the docs pages where we can update much more easily

File: enterprise/README.md (hunk 1 above)
Comment (alistair-RG) [reply enterprise/README.md]: Yes it should !

File: enterprise/README.md (hunk 2 above)
Comment (alistair-RG) [reply enterprise/README.md:5]: Reverted it back

File: enterprise/README.md (hunk 3 above)
Comment (alistair-RG) [reply enterprise/README.md]: Decided to put it back the way it was and just make the changes Adam asked for originally. We can make further changes later !

File: enterprise/README.md (hunk 1 above)
Comment (alistair-RG) [reply enterprise/README.md]: Reverted it anyway - needs TLC later

File: enterprise/README.md (hunk 2 above)
Comment (alistair-RG) [reply enterprise/README.md:5]: Actually it is outdated cruft (it's a badge from when we were using dockerhub to build the images - not doing this anymore)

File: enterprise/README.md [hunk 4]
```diff
@@ -1,33 +1,19 @@
...
-## Supported Tags
-
-The following tags are officially supported:
-
-- [`9.2.1`, `9.2`, `9`, `latest` (*Dockerfile*)](https://github.com/redgate/flyway-docker/blob/master/Dockerfile)
-- [`9.2.1-alpine`, `9.2-alpine`, `9-alpine`, `latest-alpine` (*alpine/Dockerfile*)](https://github.com/redgate/flyway-docker/blob/master/alpine/Dockerfile)
-- [`9.2.1-azure`, `9.2-azure`, `9-azure`, `latest-azure` (*azure/Dockerfile*)](https://github.com/redgate/flyway-docker/blob/master/azure/Dockerfile)
-
-The **redgate/flyway:\*-azure** images *only* support alpine versions.
+These images work across the complete range of Flyway editions, including Community, Teams and Enterprise, as well as providing compatibility with [Flyway Pipelines](https://flyway.red-gate.com/pipelines). Flyway Pipelines will help you gain centralized visibility of the state of your database deployments across projects so you can see what has been deployed, when and where for easy tracking.
```
Comment (piers-williams) [inline enterprise/README.md]: > complete range of Flyway editions

Except OSS - is there a suitable way to make this sound good while being clear that it is the proprietary, non-oss flyway? 

One idea:

```suggestion
These images work across the complete range of Redgate Flyway editions, including Community, Teams and Enterprise, as well as providing compatibility with [Flyway Pipelines](https://flyway.red-gate.com/pipelines). Flyway Pipelines will help you gain centralized visibility of the state of your database deployments across projects so you can see what has been deployed, when and where for easy tracking.
```

File: enterprise/README.md [hunk 5]
```diff
@@ -1,33 +1,19 @@
...
-`/flyway/drivers` | Directory containing the [JDBC driver for your database](https://flywaydb.org/documentation/usage/commandline/#jdbc-drivers)
-`/flyway/sql`     | The SQL files that you want Flyway to use (for [SQL-based migrations](https://flywaydb.org/documentation/concepts/migrations#sql-based-migrations))
-`/flyway/jars`    | The jars files that you want Flyway to use (for [Java-based migrations](https://flywaydb.org/documentation/concepts/migrations#java-based-migrations))
+`/flyway/conf`    | Directory containing a [configuration file](https://documentation.red-gate.com/fd/configuration-files-224003079.html)
+`/flyway/drivers` | Directory containing the [JDBC driver for your database](https://documentation.red-gate.com/fd/command-line-184127404.html)
+`/flyway/sql`     | The SQL files that you want Flyway to use (for [SQL-based migrations](https://documentation.red-gate.com/fd/migrations-184127470.html))
+`/flyway/jars`    | The jars files that you want Flyway to use (for [Java-based migrations](https://documentation.red-gate.com/fd/migrations-184127470.html))
```
Comment (piers-williams) [inline enterprise/README.md]:
```suggestion
`/flyway/jars`    | The jar files that you want Flyway to use (for [Java-based migrations](https://documentation.red-gate.com/fd/migrations-184127470.html))
```

File: enterprise/README.md [hunk 6]
```diff
@@ -46,6 +32,21 @@ agent job requirements. As it does not define an entrypoint, you need to explici
...
+
+To learn more about Flyway Pipelines you can access our [official documentation](https://documentation.red-gate.com/fd/introducing-flyway-pipelines-251363987.html)
+
+You can access the Flyway Pipelines service here:  https://flyway.red-gate.com/ 
+
+To use Flyway Pipelines, you will need a [Personal Access Token](https://documentation.red-gate.com/fd/personal-access-tokens-251363983.html). 
+
+### Example 
+
+To use the Flyway service on your image, include the following:
```
Comment (piers-williams) [inline enterprise/README.md]:
```suggestion
To use the Flyway service within this image, include the following: 
```

File: enterprise/README.md (hunk 4 above)
Comment (piers-williams) [reply enterprise/README.md]: Though I'm not so sure about this - it looks then like Redgate is distancing itself from the OSS edition

Comment (alistair-RG) [reply enterprise/README.md]: I think your suggestion is valid - they are different things and we don't want to confuse them.

**Pull request merged**